crates/bcr-wdc-swap-service/Cargo.toml

```diff
@@ -4,3 +4,23 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
+anyhow.workspace = true
+async-trait.workspace = true
+axum.workspace = true
+bcr-wdc-keys = { path = "../bcr-wdc-keys", features = ["persistence"] }
+cashu.workspace = true
+chrono.workspace = true
+config = {version = "0.15"}
+env_logger = {version = "0.11"}
```

Comment:
I think config and env_logger can go to workspace deps - they are also used in bcr-wdc-quote-service